pt/lessons/basics/collections.md

```diff
@@ -128,7 +128,7 @@ iex> map["hello"]
 :world
 ```
 
-Com em Elixir 1.2, variáveis são permitidas como chaves do mapa:
+No Elixir 1.2, variáveis são permitidas como chaves do mapa:
```

Comment:
Don't like to specify language gender, because Elixir is "a linguagem Elixir" I would say that  "Em Elixir" maybe is better. What do you think?